output_template/python/usb_camera_demo.py

```diff
@@ -225,6 +254,65 @@ def run_classification(config):
 
     cv2.destroyAllWindows()
 
+    
+def run_sementic_segmentation(config):
+    camera_width = 320
+    camera_height = 240
+    window_name = "Segmentation Demo"
+
+    vc = cv2.VideoCapture(0)
+    if not vc.isOpened():
+        print("VideoCapture failed")
+    vc.set(cv2.cv.CV_CAP_PROP_FRAME_WIDTH, camera_width)
+    vc.set(cv2.cv.CV_CAP_PROP_FRAME_HEIGHT, camera_height)
+    vc.set(cv2.cv.CV_CAP_PROP_FPS, 15)
+
+    pool = Pool(processes=1)
+    result = None
+    fps = 1.0
+
+    q_save = Queue()
+    q_show = Queue()
+
+    grabbed, camera_img = vc.read()
+    camera_img = cv2.resize(camera_img, (480, 360))
```

Comment:
I think the image not need to be resized in here, because `run_inference` apply resize as pre process.